crates/forge/src/main.rs

```diff
@@ -60,6 +65,14 @@ fn validate_fuzzer_runs_value(val: &str) -> Result<u32> {
     Ok(parsed_val)
 }
 
+fn clean_cache(package_root: &Utf8PathBuf) -> Result<()> {
```

Comment:
It is `workspace_root`, the name in `main_execution` is missleading. Changed it in #783 too